_pages/en_US/include/install-boot9strap-b9stool.txt

```diff
@@ -1,10 +1,11 @@
 If you would prefer a visual guide to this section, one is available [here](https://zoogie.github.io/web/flipnote_directions/).
 {: .notice--info}
 
-In this section, you will perform a series of very specific steps within Flipnote Studio that, when performed correctly, will launch the boot9strap (custom firmware) installer.
+In this section, you will perform a series of very specific steps within Flipnote Studio that, when performed correctly, will launch the boot9strap (custom firmware) installer B9STool.
```

Comment:
Add comma before B9STool

alternatively, `will launch B9STool, the boot9strap (custom firmware) installer.`

Pick whichever one you like